### DISCUSSION TITLE
Add icon for Engagement Intelligence Platform (EIP) product

### DISCUSSION THREAD

**Discussion by PabloFNK** [twilio-labs/paste#971] [Request: Icon] [answered]
On the transition from Console to Console.Zen we were developing a new product called Engagement Intelligence Platform (EIP) and Console.Zen forces to have the product icon on Paste (previously we were not taking it from Paste).

So we would like to add the icon to the list of product icons.

I attach a zip with the icon, it is probable that the icon could be updated in the middle or far future but we need one for Console.Zen now so we provide the one we are using on Console.

[EIP-Icon.svg.zip](https://github.com/twilio-labs/paste/files/5596286/EIP-Icon.svg.zip)

**Comment by richbachman** [comment]
Hi @PabloFNK 

Question about the icon. Do you know where it came from, and if it was approved for use by brand?

If it was approved, we're happy to add a ticket to our backlog for this.

If you need the icon added on a defined timeline, we would love for you to contribute using the guidelines here: https://paste.twilio.design/icons/how-to-add-an-icon/ 

Let us know if you have any questions!

**Comment by PabloFNK** [comment]
Hi @richbachman for some reason I thought that the product icons were following a different process. I will follow the process then.

About the questions about approval I will share this discussion with the designer as I don't know the answer.

**Comment by PabloFNK** [comment]
According to mi designer:

```
half approved. it has been shown to the brand lead but we still need an official logo from them. either a cleaned up version or another one that they will provide
```

As we don't want to wait for this as Console.Zen is already a reallity, I will follow the process to add the icon on our own and ask for update when we have the definitive one.

**Comment by PabloFNK** [comment, marked as answer]
PR to add the icon created: https://github.com/twilio-labs/paste/pull/978
